func/python/pyinit.c

```diff
@@ -126,8 +126,9 @@ void setUpPyNumpy()
     fprintf(devNull, "%Le\n", (long double)strtoull("1e100", NULL, 0));
 
     // Strings
-    FORCE_LINK(strpbrk);
+    FORCE_LINK(strcat);
     FORCE_LINK(strcspn);
+    FORCE_LINK(strpbrk);
```

Comment:
This whole file is there to ensure these functions are compiled into the static binary, so this is just adding a couple that were missing.